src/App.vue

```diff
@@ -1,27 +1,28 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
+  <!-- <img alt="Vue logo" src="./assets/logo.png"> -->
   <!-- <HelloWorld msg="Welcome to Your Vue.js App" /> -->
-  <!-- <header class="header" v-bind:title="title"></header> -->
-  <Footer class="footer" v-bind:message="footerMessage"></Footer>
+  <Header class="header" v-bind:title="title"></Header>
   <banner class="banner" v-bind:bannerMessage="messageToDisplay" v-bind:bannerType="messageType"></banner>
-  <weather>
+  <Search class="weather-search" v-on:search-city="searchCity"></Search>
```

Comment:
`class="weather-search"` same as above default classes should be used inside component itself and if any extra classes needed we can use props